hw3/01-characters.js

```diff
@@ -1,4 +1,27 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const flexbox = document.getElementById('cardbox');
+
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    // eslint-disable-next-line no-console
+    console.log(data);
+    const imageURL = 'https://thronesapi.com/assets/images/';
+    data.forEach((character) => {
+      // Extract needed data from response
+      const name = character.fullName;
+      const { title } = character;
+      const image = `${imageURL}${character.image}`;
+      // Create card element
+      const card = document.createElement('div');
+      card.classList.add('card', 'align-items-center', 'mx-auto');
+      card.innerHTML = `
+            <img src="${image}" alt="picture of ${name}" class="card-img-top mb-3">
+            <p><strong>${name}</strong></h2>
+            <p>${title}</p>
+            `;
+      flexbox.appendChild(card);
```

Comment:
Using innerHTML and hardcoding html to build out elements like this is not recommended, it will not be easy to maintain as "card" grows and changes.

Reply:
Style suggestion noted - I don't have the time to redesign the cards right now, though.